dojo/fixtures/test_type.json

```diff
@@ -691,6 +691,12 @@
         },
         "model": "dojo.test_type",
         "pk": 190
+    },
+    {
+        "fields": {
+            "name": "OssIndex DevAudit scan result importer"
```

Comment:
There is some inconsitancies in the PR for the values of parser identifier:
* `OssIndex Devaudit SCA Scan`
* `OssIndex DevAudit scan result importer`
* `OssIndex Devaudit SCA Scan Results`

Those 3 values should be the same.

Reply:
Fixed all three locations using "OssIndex Devaudit SCA Scan Importer"